src/test/java/picard/vcf/filter/TestFilterVcf.java

```diff
@@ -180,6 +178,23 @@ File testFiltering(final File vcf, final String outputExtension, final double mi
         return out;
     }
 
+    private File createTempVcfFileAndDeleteItAndItsIndexFileOnExit(final String prefix, final String suffix) throws IOException {
```

Comment:
that's quite a name..I think that temporary might be enough indication that the file will be deleted. I'd go with createTemporaryIndexedVcfFile() and you can javadoc it to explain.